grails-app/taglib/org/pih/warehouse/FormatTagLib.groovy

```diff
@@ -85,18 +69,17 @@ class FormatTagLib {
      * 				   if the locale attribute is specified, but set to "null", the "default" name is returned
      *
      * Currently simply displays the localized name of the product
+     *
+     * Output from this method will be HTML-escaped, which is always what you want
+     * when calling <format:product> but may not be when calling format.product().
      */
     def product = { attrs ->
         if (attrs.product != null) {
             // use the locale specified in the tag if it exists, otherwise use the user locale if it exists, otherwise use the system default locale
             // (note that we explicitly do a containsKey test because it is possible that the locale attribute has been specified but has been set to null--which means show the default locale)
             Locale defaultLocale = new Locale(grailsApplication.config.openboxes.locale.defaultLocale)
             Locale locale = attrs.containsKey('locale') ? attrs.locale : session?.user?.locale ?: defaultLocale
-            def value = ""
-            value += LocalizationUtil.getLocalizedString(attrs.product.name.encodeAsHTML(), locale)
-
-            // default format is to display the localized name of the product
-            out << value
+            out << LocalizationUtil.getLocalizedString(attrs.product.name, locale).encodeAsHTML()
```

Comment:
`LocalizationUtil.getLocalizedString()` won't find matches for an escaped string.